docs/reference/esql/esql-index-options.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[esql-index-options]]
+=== {esql} index options
+
+++++
+<titleabbrev>Index options</titleabbrev>
+++++
+
+Similar to the <<search-search-api-query-params,Search API>>, the `OPTIONS`
+directive of the <<esql-from,FROM>> command allows configuring the way {esql}
+accesses the data to be queried. The argument passed to this directive is a
+comma-separated list of option-name-value pairs, with the option name and the
+corresponding value quoted.
+
+[source,esql]
+----
+FROM index_pattern [OPTIONS "option1"="value1"[,...[,"optionN"="valueN"]]]
+----
+
+These options can only be provided part of a <<esql-from,FROM>> command, and
+they apply to all the indices provided or matched by an index pattern.
+
+The currently supported options are:
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=allow-no-indices]
++
+Defaults to `true`.
+
+include::{es-repo-dir}/rest-api/common-parms.asciidoc[tag=index-ignore-unavailable]
+
+include::{es-repo-dir}/search/search.asciidoc[tag=search-preference]
```

Comment:
> is the linked information still correct given that ESQL executes queries distributed by design?

What would be a "versus" for this execution type, which the options wouldn't apply to?

My understanding is that we resolve the shards similarly to how _search does. So these settings should have the same effect:
* `_only_local`: yes, coordinator node only (and it could return nothing, because of that, just like when providing a `_shards` with non-matching shard IDs).
* `_local`: as explained, local node, if possible, "default method" otherwise.
* `_prefer_nodes `: my understanding of the "if possible" node is that it'll try the provided nodes (whatever types they are) and regress to the "default method" otherwise.